.travis.yml

```diff
@@ -0,0 +1,46 @@
+dist: xenial
+sudo: required
+language: java
+jdk:
+  - openjdk11
+
+services:
+  - docker
+
+cache:
+  directories:
+    - $HOME/.m2
+# the cache can grow constantly
+before_cache:
+  - rm -rf $HOME/.m2/repository/org/alfresco/alfresco-greenmail
+
+branches:
+  only:
+    - master
+    - /support\/.*/
+    - /feature\/.*/
+    - /fix\/.*/
```

Comment:
I don't think it is required to build all these branches. Especially not for this project. Please leave "master" and "support", the rest will be built as PRs.

Reply:
Sure.